pwa-devdocs/README.md

```diff
@@ -76,8 +76,8 @@ If you want to contribute, please review the [contribution guidelines] and follo
 
 This project is licensed under OSL-3.0 - see the [LICENSE.txt] file for details.
 
-[contribution guidelines]: ../../.github/CONTRIBUTING.md
-[code of conduct]: ../../.github/CODE_OF_CONDUCT.md
+[contribution guidelines]: https://magento.github.io/pwa-studio/technologies/contribute/
+[code of conduct]: https://github.com/magento/pwa-studio/blob/master/.github/CODE_OF_CONDUCT.md
```

Comment:
```suggestion
[code of conduct]: https://github.com/magento/pwa-studio/blob/develop/.github/CODE_OF_CONDUCT.md
```

Reply:
**fixed**

Reply:
@sirugh , could you please review?

Thank you!